apps/vercel/src/locations/ConfigScreen.tsx

```diff
@@ -103,6 +140,22 @@ const ConfigScreen = () => {
                 to create an access token in the Vercel dashboard.
               </HelpText>
             </FormControl>
+            <Box style={{ width: '100%' }}>
```

Comment:
nit: I know a lot of things are in progress, but perhaps these inline styles can be in separate style files in the future?